apps/schools/domains/common/components/containers/BaseLayoutComponents/BaseLayout/index.tsx

```diff
@@ -12,17 +12,68 @@ import { IBaseLayoutProps } from './interfaces'
+import Image from 'next/image'
+import ExclamationCircleOutlined from '@public/icons/ExclamationCircleOutlined.svg'
+import { Button } from '@domains/common/components/button'
+import { handleChangeStatusInvitation } from '@domains/common/handlers/changeStatusInvitation'
+import { useChangeStatusMutation } from '@domains/query/redux/queryApi'
+import { useGetInvitationsQuery } from '@domains/employee/redux/employeeApi'
 
 const { Content, Sider } = Layout
 
 export const BaseLayout: React.FC<IBaseLayoutProps> = (props) => {
     const { isCollapsed, toggleCollapsed } = useLayoutContext()
+    const { data: invitations, refetch } = useGetInvitationsQuery({})
+    const [mutation] = useChangeStatusMutation()
 
     const { children } = props
 
+    invitations?.results.map((invite) => console.log(invite))
+
     return (
         <>
             <Layout className={styles.placement}>
+                <div className={styles.invitationsContainer}>
+                    {invitations?.results.map((invite) => {
+                        if (invite.status === QueryStatuses.SENT || invite.status === QueryStatuses.IN_PROGRESS) {
+                            return (
+                                <div className={styles.inviteContainer} key={invite.id}>
+                                    <div className={styles.leftContainer}>
+                                        <Image src={ExclamationCircleOutlined} alt={'exclamation circle outlined'} />
+                                        <div>Вас пригласили в организацию “{invite.sender.name}”</div>
+                                    </div>
+                                    <div className={styles.rightContainer}>
+                                        <Button
+                                            className={styles.acceptButton}
+                                            onClick={() =>
+                                                handleChangeStatusInvitation(
+                                                    mutation,
+                                                    invite.id,
+                                                    QueryStatuses.ACCEPTED,
+                                                ).then(refetch)
+                                            }
+                                        >
+                                            Принять
+                                        </Button>
+                                        <Button
+                                            className={styles.rejectedButton}
+                                            onClick={() =>
+                                                handleChangeStatusInvitation(
+                                                    mutation,
+                                                    invite.id,
+                                                    QueryStatuses.DECLINED,
+                                                ).then(refetch)
```

Comment:
it is ok for now but can we change our redux store without making additional request?
it would be cooler and wiser

Reply:
should we also refetch organizations to display it in select?

Reply:
organization change should also trigger refetch all other requests that is related to organization